cluster/src/fpindexcluster/proxy_service.py

```
@@ -0,0 +1,267 @@
+                logger.error(f"Failed to process change {change}: {e}")
+                results.append({"error": str(e), "change": change})
+        
+        return web.json_response({"results": results})
+    
+    async def search(self, request: web.Request) -> web.Response:
+        """Handle search requests - forward to actual fpindex"""
+        index_name = request.match_info["index"]
+        
+        try:
+            query_data = await request.json()
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)